### PR TITLE
Do not require file or directory for ghost entries (#56).

### LINKED ISSUE #56: ghost entries
I would like to specify a ghost entry, but the plugin requires a file, directory or collect element to be specified for it anyway, which does not seem to make much sense to me.

For example:
```
  <entry>
    <name>/var/log/myapp.log</name>
    <ghost>true</ghost>
  </entry>
```

Yields:
[ERROR] Failed to execute goal de.dentrassi.maven:rpm:1.6.0:rpm (default) on project myapp: Exactly one of 'file', 'directory', 'linkTo' or 'collect' must be specified.

If adding a file element referring to a non-existing file to the above entry, will lead to the following trace:
```
    at org.eclipse.packager.rpm.build.RpmBuilder$1.addFile (RpmBuilder.java:1355)
    at de.dentrassi.rpm.builder.RpmMojo.fillFromEntryFile (RpmMojo.java:1118)
    at de.dentrassi.rpm.builder.RpmMojo.fillFromEntry (RpmMojo.java:1094)
    at de.dentrassi.rpm.builder.RpmMojo.fillPayload (RpmMojo.java:1063)
    at de.dentrassi.rpm.builder.RpmMojo.execute (RpmMojo.java:787)
```

Using collect element that does not find a file, will not lead to an error, but the produced RPM will not know about the ghost file either.
Using skip will not lead to an error in either case, but the produced RPM will not know about the ghost file either.

So it seems that currently the only way to refer to an existing empty file, which seems to be cumbersome and not intuitive.

### PR DESCRIPTION
**Pull request by dranuhl**
This pull request resolves #56.

Done a few tests with RPM 4.11.x, which does not list any as ghost marked files anymore if the not really documented option ``--noghost`` is been given.

### CODE REVIEW

**Comment by ctron** [comment]
That looks like a simple approach! :+1: 

I would appreciate it however, if you could add a new integration test for this one.

File: src/it/test16-ghost/pom.xml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+						<goals>
+							<goal>rpm</goal>
+						</goals>
+						<configuration>
+							<group>Application/Misc</group>
+
+							<outputFileName>test16.rpm</outputFileName>
+
+							<entries>
+								<entry>
```
Comment (ctron) [inline src/it/test16-ghost/pom.xml:55]: Should that entry contain `<ghost>true</ghost>`?

**Comment by ctron** [comment]
I updated the master branch to enable GitHub actions also for PRs. And dropped travis. I guess you need to rebase though, if you want to use it.

**Comment by dranuhl** [comment]
So I have completed my own development testing for the unit and integration test additions (should have probably switched to draft PR in the meantime, sorry.)

Added 2 changesets:
1. added unit tests for modified PackageEntry class.
2. added integration test for ghost entry handling using native RPM command.

File: src/it/test16-ghost/pom.xml (hunk 1 above)
Comment (dranuhl) [reply src/it/test16-ghost/pom.xml:55]: Fixed in latest version.

**Comment by ctron** [comment]
Looks good to me … do you want me to merge and release?

**Comment by dranuhl** [comment]
Sure.

**Comment by ctron** [comment]
Release as part of 1.7.0.

Many thanks!

**Pull request merged**